shell/src/main/java/alluxio/cli/fsadmin/command/BackupCommand.java

```diff
@@ -56,15 +58,13 @@ public Options getOptions() {
   @Override
   public int run(CommandLine cl) throws IOException {
     String[] args = cl.getArgs();
-    String dir;
-    if (args.length < 1) {
-      dir = null;
-    } else {
-      dir = args[0];
+    Builder opts = BackupPOptions.newBuilder();
+    if (args.length == 1) {
```

Comment:
Slight difference in functionality, do you mind adding a check for `if args > 1` and print the help or something? Otherwise users incorrectly using the args will silently see different behavior

Reply:
The number of arguments validation is already done by the `validateArgs` method - in `run` we can assume that the correct number of arguments have been given